packages/conventional-changelog-angular/test/test.js

```diff
@@ -1,334 +1,337 @@
-'use strict';
-var execSync = require('child_process').execSync;
-var conventionalChangelogCore = require('conventional-changelog-core');
-var preset = require('../');
-var expect = require('chai').expect;
-var gitDummyCommit = require('git-dummy-commit');
-var shell = require('shelljs');
-var through = require('through2');
-var betterThanBefore = require('better-than-before')();
-var preparing = betterThanBefore.preparing;
+'use strict'
+var conventionalChangelogCore = require('conventional-changelog-core')
```

Comment:
I'm a little disappointed `standard` doesn't encourage the use of `const`, or even outright ban the use of `var`. Oh well. I'll probably switch these all over later.

Reply:
If need be, I can add the [no-var](https://eslint.org/docs/rules/no-var) rule to the config.

Reply:
No, let's wait :smile: 

This change is to just adopt the `standard` rules for now, and then we can discuss enforcing `const` across all the `conventional-changelog` related repositories.